book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author, :date, :person
+  attr_reader :rentals
```

Comment:
- [ ] Getters and Setters for @rentals should be present, kindly add rentals to `attr_accessor` instead of `attr_reader`